openmc/deplete/abc.py

```diff
@@ -842,7 +845,7 @@ def _get_bos_data_from_restart(self, step_index, source_rate, bos_conc):
         k = ufloat(res.k[0, 0], res.k[0, 1])
 
         # Scale reaction rates by ratio of source rates
-        rates *= source_rate / res.source_rate[0]
+        rates *= source_rate / res.source_rate
```

Comment:
Is this change because the step results only contain one time step now so they have a one-dimension smaller array? In that the entire `source_rate` attribute is from the current step?

Reply:
The .h5 file still stores the source rate for each operator call within a timestep, but the source rate is always the same so when `StepResult` is populated, I changed it so that it only gets the first value. I think this better reflects what a user would reflect so they can easily query "what was the source rate for this step?" and not get an array of values.

Reply:
That makes a lot more sense. Nice!